### DISCUSSION TITLE
Empty content of community/discussions on Android

### DISCUSSION THREAD

**Discussion by adoyle-h** [community/community#48983] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Open https://github.com/community/community/discussions/ on Github Mobile (Android), it shows that.

<img width="300px" src="https://user-images.githubusercontent.com/1998490/222445558-5e2a4cec-ffb1-4bf2-8649-bea64da33820.jpg"></img>

Version: v1.101.0 (706)

**Comment by eliperkins** [comment, marked as answer]
Hey @adoyle-h , thanks for submitting this bug! There should definitely be some content in that list view. 😅  I've filed an issue in our internal tracking issue for this, and I'll follow up with our engineering team to get this fixed. I'll keep you posted!

**Reply by adoyle-h** [reply]
It works now. Current Github Mobile version is v1.103.0 (708).

<img width="200px" src="https://user-images.githubusercontent.com/1998490/225853861-d3a48889-ccf5-42fd-932d-b13a08c6751f.jpg">

**Comment by aconsuegra** [comment]
Hi @adoyle-h, may I ask you which version of Android are you running and the device that you are using? Thanks!

**Reply by adoyle-h** [reply]
I'm using [Xiaomi Mi 10 Ultra](https://www.gsmarena.com/xiaomi_mi_10_ultra-10361.php) with MIUI 13.0.5. Current Android version is 12.

**Comment by Chaminade1** [comment]
Updage your BIOS or all the system
